### ISSUE TITLE
Problem with package.json that has not only one name

### ISSUE THREAD

**Issue by drefixs** [bpkg/bpkg#106]
i have the next packege.json file
```
{
  "name": "bash-oo-framework",
  "version": "2.1.0",
  "description": "Bash Infinity is a modern boilerplate ",
  "repo": "niieani/bash-oo-framework",
  "files": [
    "lib/Array/Contains.sh",
    "lib/Array/Intersect.sh",
    "lib/Array/List.sh",
    "lib/Array/Reverse.sh",
    "lib/String/GetSpaces.sh",
    "lib/String/IsNumber.sh",
    "lib/String/SanitizeForVariable.sh",
    "lib/String/SlashReplacement.sh",
    "lib/String/UUID.sh",
    "lib/TypePrimitives/array.sh",
    "lib/TypePrimitives/boolean.sh",
    "lib/TypePrimitives/integer.sh",
    "lib/TypePrimitives/map.sh",
    "lib/TypePrimitives/string.sh",
    "lib/UI/Color.sh",
    "lib/UI/Color.var.sh",
    "lib/UI/Console.sh",
    "lib/UI/Cursor.sh",
    "lib/util/bash4.sh",
    "lib/util/class.sh",
    "lib/util/command.sh",
    "lib/util/exception.sh",
    "lib/util/log.sh",
    "lib/util/namedParameters.sh",
    "lib/util/pipe.sh",
    "lib/util/test.sh",
    "lib/util/tryCatch.sh",
    "lib/util/type.sh",
    "lib/util/variable.sh",
    "lib/oo-bootstrap.sh"
  ]
}
```

And i have install like this
```
bpkg install drefixs/bash-oo-framework

    info: Install /drefixs/bash-oo-framework/master from remote https://raw.githubusercontent.com [https://github.com]
    info: Install dependencies for lib/util/namedParameters.sh
bash-oo-framework
EXPECTED value GOT ]
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/Array/Contains.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/Array/Contains.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/Array/Intersect.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/Array/Intersect.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/Array/List.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/Array/List.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/Array/Reverse.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/Array/Reverse.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/String/GetSpaces.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/String/GetSpaces.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/String/IsNumber.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/String/IsNumber.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/String/SanitizeForVariable.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/String/SanitizeForVariable.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/String/SlashReplacement.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/String/SlashReplacement.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/String/UUID.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/String/UUID.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/TypePrimitives/array.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/TypePrimitives/array.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/TypePrimitives/boolean.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/TypePrimitives/boolean.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/TypePrimitives/integer.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/TypePrimitives/integer.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/TypePrimitives/map.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/TypePrimitives/map.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/TypePrimitives/string.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/TypePrimitives/string.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/UI/Color.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/UI/Color.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/UI/Color.var.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/UI/Color.var.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/UI/Console.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/UI/Console.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/UI/Cursor.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/UI/Cursor.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/util/bash4.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/util/bash4.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/util/class.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/util/class.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/util/command.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/util/command.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/util/exception.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/util/exception.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/util/log.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/util/log.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/util/namedParameters.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/util/namedParameters.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/util/pipe.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/util/pipe.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/util/test.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/util/test.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/util/tryCatch.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/util/tryCatch.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/util/type.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/util/type.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/util/variable.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/util/variable.sh
    fetch: https://raw.githubusercontent.com/drefixs/bash-oo-framework/master/lib/oo-bootstrap.sh
    write: /Users/drefixs/projects/st-kube/deps/lib/util/namedParameters.sh
bash-oo-framework/lib/oo-bootstrap.sh
```

**Comment by drefixs** [comment]
pls review and merge this fix to solve this bug #105 
:)

**Comment by Cj-bc** [comment]
Hello, I've got the same problem so I'm happy to your PR ;)
I hope this will be merged sooner...

**Comment by Potherca** [comment]
Closing issue as it has been resolved by MR #105

Release as part of [v0.3.1](https://github.com/bpkg/bpkg/releases/tag/0.3.1)
